controllers/activemqartemis_reconciler.go

```diff
@@ -846,6 +848,9 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) configureConsoleExposure(custom
 			}
 		}
 	}
+	if len(currentStatefulSet.Spec.Template.Spec.Containers) > 0 {
+		currentStatefulSet.Spec.Template.Spec.Containers[0].Ports = containerPorts
```

Comment:
I think this may step on:
	containerPorts := MakeContainerPorts(customResource)
when the jolokia agent is in play.

We may need to add this logic to make container ports or append

Reply:
yes good point. Will update to append and include it in test.